### DISCUSSION TITLE
Back-in-stock notifications

### DISCUSSION THREAD

**Discussion by mewa** [swellstores/community#424] [Show and tell]
Hello everyone,

We created an integration to handle back-in-stock notifications for our store.

If you'd like to integrate it with your Swell store as well just let me know or sign up [here](https://swelltoolkit.com/stock-alerts?utm_source=github&utm_medium=announcement&utm_content=link&utm_term=stock_alerts).

Thanks,
Marcin

**Comment by sags95** [comment]
Hey @mewa, this is great! Will reach out to you in an email shortly.
